### PR TITLE
Add the email capture form on several more pages

### PR DESCRIPTION
**Pull request by ThatOtherPerson**
Fixes #897

### CODE REVIEW

**Comment by KyleAMathews** [comment]
Deploy preview for *using-postcss-sass* failed.

Built with commit 99359d43e8fe094df471e97cd3efd47024011d5d

https://app.netlify.com/sites/using-postcss-sass/deploys/5b68da4d73f2cf0dd6e37082

**Comment by KyleAMathews** [comment]
Deploy preview for *gatsbyjs* failed.

Built with commit 99359d43e8fe094df471e97cd3efd47024011d5d

https://app.netlify.com/sites/gatsbyjs/deploys/5b68da4c73f2cf0dd6e37076

**Comment by calcsam** [comment]
Looks great, thanks Cassie!

File: www/src/pages/index.js [hunk 1]
```diff
@@ -198,12 +199,11 @@ class IndexRoute extends React.Component {
...
                         />
                       ))}
-                      <CtaButton
-                        to="/blog/"
-                        overrideCSS={{ marginBottom: rhythm(2) }}
-                      >
-                        Read More
-                      </CtaButton>
+                      <CtaButton to="/blog/">Read More</CtaButton>
+                      <EmailCaptureForm
```
Comment (calcsam) [inline www/src/pages/index.js]: This should be below the Getting Started, on top of the blog

**Pull request merged**